german/GUI_Automation/src/main/java/at05ui/core/selenium/webdrivers/Chrome.java

```diff
@@ -0,0 +1,28 @@
+package at05ui.core.selenium.webdrivers;
+
+import io.github.bonigarcia.wdm.ChromeDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+class Chrome implements IDriver {
+
+    private static Chrome instance=null;
+    private WebDriver driver;
+    private WebDriverWait wait;
+
+    public static Chrome getInstance() {
+        if (instance == null || instance.driver == null ) {
+            instance = new Chrome();
+        }
+        return instance;
```

Comment:
Why are you using this? it is not necessary.

Reply:
Yes, I will remove it